cg/cli/workflow/commands.py

```diff
@@ -383,3 +384,22 @@ def microsalt_past_run_dirs(
     context.invoke(
         past_run_dirs, skip_confirmation=skip_confirmation, dry_run=dry_run, before_str=before_str
     )
+
+
+@click.command("tower-past-run-dirs")
+@SKIP_CONFIRMATION
+@ARGUMENT_BEFORE_STR
+@click.option(
+    "-w",
+    "--workflow",
+    "workflow",
+    help="Only cleans analyses from the specified workflow, if provided.",
+)
+@click.pass_context
+def tower_past_run_dirs(
```

Comment:
A bit uncertain of how this should be written? Should it be one command with workflow as a parameter or should it be several commands? The workflow is needed for the analysis api instantiation.

Reply:
I think one command with the workflow as parameter is better

Reply:
Rnafusion command should be removed as redundant with tower now

Reply:
Good catch! If there is automation running I would prefer rewriting the services first and remove in a later PR, though, so as to not break anything.